status-panels-lib/README.md

```diff
@@ -0,0 +1,161 @@
+# Status panels lib
+
+This status panels lib can be used to add a status panel row for integration dashboards using [grafonnet](https://github.com/grafana/grafonnet).
+
+The status panels provide the following information for an integration.
+
+- Metrics status
+  - Shows if the agent is sending metrics for this integration
+  - Shows the latest timestamp the metrics were received for this integration
+- Logs status
+  - Shows if the agent is sending logs for this integration
+  - Shows the latest timestamp the logs were received for this integration
+- Integration version
+  - Shows the current installed version of the integration
+
+## Import
+
+```sh
+jb init
+jb install https://github.com/grafana/jsonnet-libs/status-panels-lib
+```
+
+# Usage
+
+## Metrics
+
+```
+(statusPanels.new(
+  'Integration Status',
+  type='metrics',
+  statusPanelsQueryMetrics='up{job=~"$job"}',
```

Comment:
Are you still planning to use the metricsCheckQuery and logsCheckQuery fields?
I think we can gain using them, since the `up` metrics can give false positives for exporters that are up and running but actually not collecting data from SUO for varios reasons.

Reply:
Just gave a bit more details to the `statusPanelsQueryMetrics` docs. Think this covers.